model/shared/src/main/scala/explore/model/syntax/package.scala

```diff
@@ -3,29 +3,30 @@
 
 package explore.model.syntax
 
+import cats.syntax.all._
 import explore.model.enum.PosAngleOptions
 import lucuma.core.math.Angle
-import lucuma.core.model.PosAngle
+import lucuma.core.model.PosAngleConstraint
 
 object all {
 
   implicit class PosAngleOptionsOps(val pa: PosAngleOptions) extends AnyVal {
-    def toPosAngle(a: Angle): PosAngle = pa match {
-      case PosAngleOptions.Fixed               => PosAngle.Fixed(a)
-      case PosAngleOptions.AllowFlip           => PosAngle.AllowFlip(a)
-      case PosAngleOptions.AverageParallactic  => PosAngle.AverageParallactic
-      case PosAngleOptions.ParallacticOverride => PosAngle.ParallacticOverride(a)
-      case PosAngleOptions.Unconstrained       => PosAngle.Unconstrained
+    def toPosAngle(a: Angle): Option[PosAngleConstraint] = pa match {
+      case PosAngleOptions.Fixed               => PosAngleConstraint.Fixed(a).some
+      case PosAngleOptions.AllowFlip           => PosAngleConstraint.AllowFlip(a).some
+      case PosAngleOptions.AverageParallactic  => PosAngleConstraint.AverageParallactic.some
+      case PosAngleOptions.ParallacticOverride => PosAngleConstraint.ParallacticOverride(a).some
+      case PosAngleOptions.Unconstrained       => none
     }
   }
 
-  implicit class PosAngleOps(val pa: PosAngle) extends AnyVal {
+  implicit class PosAngleOps(val pa: Option[PosAngleConstraint]) extends AnyVal {
     def toPosAngleOption: PosAngleOptions = pa match {
```

Comment:
Ultra-picky comment: maybe call this `toPosAngleOptions` ? At first glance of `toPosAngleOption` I was expecting the return type to be `Option[PosAngle]`.